src/parser.rs

```diff
@@ -334,7 +334,7 @@ pub struct LocalVarDef {
 /// This is a "dense" segment, which must be fully rebuilt in order to make any change.  We may in
 /// the future have an "unpacked" segment which is used for active editing, as well as a "lazy" or
 /// "mmap" segment type for fast incremental startup.
-#[derive(Debug)]
+#[derive(Debug,Default)]
```

Comment:
I can't think of an alternative, though, since `derive()` is not accessible to custom implementation. It would be nice if there was a way to override `default()` but use the derived version in the implementation.